build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
 	compile "io.gitlab.arturbosch.detekt:detekt-rules:$detektVersion"
 	provided "org.sonarsource.sonarqube:sonar-plugin-api:$sonarVersion"
 
+	testCompile "junit:junit:4.12"
```

Comment:
Should we also follow version mentioning in dependencies as it is for other?

Reply:
Please use the gradle.properties for versions.